### PR TITLE
Edits based on single-server scenario testing

### PR DESCRIPTION
**Pull request by ManikaDhiman**

### CODE REVIEW

**Comment by Court72** [comment]
@ronmiab

After this PR is marked "ready for review," can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by Court72** [comment]
cc @dansisson

**Review by ronmiab — APPROVED** [review]
I have reviewed your changes and I think everything looks great 🙂!

**Comment by ronmiab** [comment]
@ManikaDhiman - great job on the revisions. I think you're good to go.

File: azure-stack/hci/deploy/single-server.md [hunk 1]
```diff
@@ -27,31 +27,40 @@ Currently you can't use Windows Admin Center to deploy Azure Stack HCI on a sing
...
-   Enable-ClusterStorageSpacesDirect -CacheState Disabled 
+   New-Cluster -Name $ClusterName -Node $ServerName -nostorage
+   Enable-ClusterStorageSpacesDirect -CacheState Disabled
    ```
 
-   > [!Note]
-   > The `New-Cluster` command will also require the `StaticAddress` parameter if the node is not using DHCP for its IP address assignment.  This parameter should be supplied with a new, available IP address on the node's subnet.
+   > [!NOTE]
+   > - The cluster name should not exceed 15 characters.
+   > - The `New-Cluster` command will also require the `StaticAddress` parameter if the node is not using DHCP for its IP address assignment.  This parameter should be supplied with a new, available IP address on the node's subnet.
```
Comment (sethmanheim) [inline azure-stack/hci/deploy/single-server.md:60]:
```suggestion
   > - The `New-Cluster` command also requires the `StaticAddress` parameter if the node is not using DHCP for its IP address assignment. This parameter should be supplied with a new, available IP address on the node's subnet.
```

**Comment by Jak-MS** [comment]
@ManikaDhiman - take a look at @sethmanheim's suggestions, and when you're ready, take the PR out of draft mode. After that @ronmiab can add `#sign-off`

**Pull request merged**